lib/linux_kernel/processor.c

```diff
@@ -136,9 +128,7 @@ static void processors_bitmap_copy(struct m0_bitmap *dest,
 
 /**
    Fetch NUMA node id for a given processor.
-
```

Comment:
Why is this blank line removed?

But in the new implementation for AARCH64, you have similar blank lines there.

Reply:
https://github.com/Seagate/cortx-motr/pull/653#discussion_r646693156

You removed all the blank lines here. Why?

It's fine to remove them.
But in the new implementation for AARCH64, you have similar blank lines in the comments of those functions.
Why are there blank line?
When you resolve a review comment, you'd better to say that it is rejected, or it is fixed.
This comment is resolved by you silently.
I don't know whether you've really resolved it.

Reply:
I have put back the blank line between the comment line as before in both x86 and aarch64 section. but the line no does not match exactly, as there are few line deleted at the beginning